pkg/apis/operator/v1alpha1/types.go

```diff
@@ -398,6 +401,75 @@ type KubeControllerManagerConfig struct {
 	CertificateSigningDuration *metav1.Duration `json:"certificateSigningDuration,omitempty"`
 }
 
+// Gardener contains the configuration settings for the Gardener componenets.
+type Gardener struct {
+	// ClusterIdentity is the identity of the garden cluster. This field is immutable.
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
+	ClusterIdentity string `json:"clusterIdentity"`
+	// APIServer contains configuration settings for the gardener-apiserver.
+	// +optional
+	APIServer *GardenerAPIServerConfig `json:"gardenerAPIServer,omitempty"`
+	// ControllerManager contains configuration settings for the gardener-controller-manager.
+	// +optional
+	ControllerManager *GardenerControllerManagerConfig `json:"gardenerControllerManager,omitempty"`
+	// Scheduler contains configuration settings for the gardener-scheduler.
+	// +optional
+	Scheduler *GardenerSchedulerConfig `json:"gardenerScheduler,omitempty"`
+}
+
+// GardenerAPIServerConfig contains configuration settings for the gardener-apiserver.
+type GardenerAPIServerConfig struct {
+	gardencorev1beta1.KubernetesConfig `json:",inline"`
+	// AdmissionPlugins contains the list of user-defined admission plugins (additional to those managed by Gardener),
+	// and, if desired, the corresponding configuration.
+	// +optional
+	AdmissionPlugins []gardencorev1beta1.AdmissionPlugin `json:"admissionPlugins,omitempty"`
+	// AuditConfig contains configuration settings for the audit of the kube-apiserver.
+	// +optional
+	AuditConfig *gardencorev1beta1.AuditConfig `json:"auditConfig,omitempty"`
+	// Logging contains configuration for the log level and HTTP access logs.
+	// +optional
+	Logging *gardencorev1beta1.KubeAPIServerLogging `json:"logging,omitempty"`
+	// Requests contains configuration for request-specific settings for the kube-apiserver.
+	// +optional
+	Requests *gardencorev1beta1.KubeAPIServerRequests `json:"requests,omitempty"`
+	// WatchCacheSizes contains configuration of the API server's watch cache sizes.
+	// Configuring these flags might be useful for large-scale Shoot clusters with a lot of parallel update requests
```

Comment:
Is this really about the shoot clusters, but not rather the seeds?

```suggestion
	// Configuring these flags might be useful for large-scale Garden clusters with a lot of parallel update requests
```